### PR TITLE
Add FileSystem completion

### PR DESCRIPTION
**Pull request by glacambre**

### CODE REVIEW

**Comment by bovine3dom** [comment]
This is neat. In testing, all other completions such as `open` get "Filesystem" flashing above them. Can you reproduce this?

**Comment by glacambre** [comment]
Should be fixed now.

**Comment by bovine3dom** [comment]
oh, woops, I merged via a rebase again. One day I'll remember.

**Comment by glacambre** [comment]
Sorry. I'll investigate and reopen once fixed.

**Comment by glacambre** [comment]
Problem was an unresolved promise on error. Should be fixed now.

File: src/completions/FileSystem.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <td class="value">${value}</td>
+        </tr>`
+    }
+}
+
+export class FileSystemCompletionSource extends Completions.CompletionSourceFuse {
+    public options: FileSystemCompletionOption[]
+
+    constructor(private _parent) {
+        super(["saveas"], "FileSystemCompletionSource", "FileSystem")
```
Comment (bovine3dom) [inline src/completions/FileSystem.ts]: Could you add `source` here before I forget?

Comment (glacambre) [reply src/completions/FileSystem.ts]: Done.

**Pull request merged**